provisioning/buffer/kubernetes/templates/etcd.yaml

```diff
@@ -4,9 +4,9 @@
 resources:
   requests:
     cpu: "200m"
-    memory: "768Mi"
+    memory: "2000Mi"
```

Comment:
Tu rozmyslam ci by nebolo lepsie nechat requesty ako su alebo zvysit max na 1GB, pretoze [ostatne stacky maju max memory usage od 150M do 500MB](https://app.datadoghq.eu/dashboard/n3c-jfd-8jm/buffer-service?fullscreen_end_ts=1679390488446&fullscreen_paused=false&fullscreen_section=overview&fullscreen_start_ts=1679386888446&fullscreen_widget=3082274334045506&tpl_var_env%5B0%5D=kbc-testing-azure-east-us-2&tpl_var_env%5B1%5D=kbc-us-east-1&tpl_var_env%5B2%5D=com-keboola-azure-north-europe&tpl_var_env%5B3%5D=dev-keboola-aws-eu-west-1&from_ts=1679386496072&to_ts=1679390096072&live=true). Takto sa nam tym napr zdrazia testing stacky. 
Pripadne by sa to dalo nastavovat variablne cez nejaku env stacku ETCD_MEMORY_REQUEST.

Reply:
Ostatne stacks este nemaju klieta co tam posiela `200-300req/s`, ale to by mal byt v GA normalny provoz. Dizajnovane je to na `2000req/s+`, ak tam bude na ostatnych stacks taky traffic ako teraz, tak to znamena, ze to nikto nepouziva a mozeme to vypnut.

Vratil by som sa k tomu o 2 tyzdne, mozno nam bude vsade stacit `1,2GB` alebo `1.5GB`.

Reply:
tak tie testing stacky budu mat vzdy nizky provoz (~150MB), ale suhlas, to potom ked tak zoptimaluzjeme.

Reply:
na testingu potrebujem/chcem spravit zatazove testy, ale este som sa k tomu nedostal, potom to dam kludne dole.